scripts/check-coding-style.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
-script_dir=$(dirname "$(readlink -f "$0")")
-
+READLINK=$(which greadlink)
+[ -z "$READLINK" ] && {
+	READLINK=$(which readlink)
+}
+[ -z "$1" ] && {
+	echo "USAGE: $0 FILE"
+	exit 1
+}
+CHECKSOURCE=$($READLINK -f "$1")
```

Comment:
I thought of the script as an helper to run the actual perl code-checker with the right parameters. That way, the script can check any file passed as an argument. 

As our goal is to have all files kernel coding style compliant, we can run something like 
```
find . -name "*tommyds-1.8*" -prune -name "*.c" -o -name "*.h" -exec ./scripts/check-coding-style.sh {} \;
```
in the _rtrlib_ repo-root - and add this command to the travis config.